packages/babel-preset/CHANGELOG.md

```diff
@@ -0,0 +1,175 @@
+# Changelog
+
+## [Unreleased]
+
+The babel preset is now scoped undo the `@shopify` name. This mean the name of the preset is now `@shopify/babel-preset`.
```

Comment:
We should make this more noticeable with a different header. Something like: 
## 🚨Package rename